tests/E2E/test/campaigns/high/13_employee/1_employee.js

```diff
@@ -65,7 +65,7 @@ scenario('Delete an employee', client => {
     return promise
       .then(() => client.waitForVisible(Employee.search_result))
       .then(() => client.getTextInVar(Employee.search_result,'employee_number'))
-      .then(() => tab['employee_number'] = tab['employee_number'].split('(')[1].split(')')[0])
+      .then(() => tab['employee_number'] = tab['employee_number'].match(/\d+/g)[0])
```

Comment:
hum.. this regex is too greedy and doesn't take in consideration the previous requirements `(` and `)`

Reply:
What do you think about this ?
tab['employee_number'] = /\(([^)]+)\)/.exec(tab['employee_number'])[1]

Reply:
Seems better :)

Reply:
OK done.